routes/person.js

```diff
@@ -65,3 +38,31 @@ module.exports = (elastic, config) => ({
     }
   }
 });
+
+function HTMLResponse (request, reply, elastic, config) {
+  const data = {
+    page: 'person'
+  };
+
+  elastic.get({index: 'smg', type: 'agent', id: TypeMapping.toInternal(request.params.id)}, (err, result) => {
```

Comment:
Maybe we could create a function for this elasticsearch request. It is almost the same code as in the json api handler.